src/classes/CON_DeleteContactOverride_TEST.cls

```diff
@@ -319,7 +319,214 @@ private class CON_DeleteContactOverride_TEST {
         System.assertNotEquals(null, con, 'The Contact should be retrieved from DB.');
         System.assert(con.isDeleted, 'The Contact should be deleted');
     }
+
+    /*********************************************************************************************************
+    * @description Attempts to delete contact with closed/won and org opps, verifies correct exception is thrown.
+    */
+    static testMethod void testContactDeleteExceptionWithOverride(){
+        //use a non-system account for the contact's account, so that the delete override doesn't redirect us to delete the account
+        Account actualAccount = new Account(Name = 'TEST_ACCOUNT');
+        insert actualAccount;
+
+        Contact actualContact = new Contact(LastName = 'TEST_CONTACT', AccountId = actualAccount.Id);
```

Comment:
We can simply declare them `contact`, `account`, `case` vars in this and the following test.